### PR TITLE
Update the order.

### PR DESCRIPTION
**Pull request by jj-zhang**
address issue #134

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -80,6 +80,7 @@ A higher ordinal value means that the values taken from this ConfigSource will o
 
 There are three default configuration sources as following:
 
+* A <variable name="…​" value="…​"/> element in server.xml has a default ordinal of 500.
```
Comment (gkwan-ibm) [inline README.adoc:83]: need backtick `<variable name="…​" value="…​"/>`

**Comment by gkwan-ibm** [comment]
@jj-zhang 
- [ ] change three to four for the statement: "There are three default configuration sources as following:"

**Comment by gkwan-ibm** [comment]
@Charlotte-Holt would you review the change although it is a simple statement?

File: README.adoc [hunk 2]
```diff
@@ -78,8 +78,9 @@ MicroProfile Config combines configuration properties from multiple sources, eac
 
 A higher ordinal value means that the values taken from this ConfigSource will override values from ConfigSources with a lower ordinal value.
 
-There are three default configuration sources as following:
+There are four default configuration sources as following:
```
Comment (Charlotte-Holt) [inline README.adoc:81]: I'd suggest updating this to
"The following four sources are the default configuration sources:"

File: README.adoc [hunk 3]
```diff
@@ -78,8 +78,9 @@ MicroProfile Config combines configuration properties from multiple sources, eac
...
 
+* A `<variable name="…​" value="…​"/>` element in server.xml has a default ordinal of 500.
```
Comment (Charlotte-Holt) [inline README.adoc:83]: Suggestion:
"A `<variable name="..." value="..."/>` element in the `server.xml` file has a default ordinal of 500."

**Comment by gkwan-ibm** [comment]
@jj-zhang would you fix https://github.com/OpenLiberty/guide-microprofile-config/pull/135#discussion_r410455419?

**Comment by gkwan-ibm** [comment]
@Charlotte-Holt Darren addressed your feedback, would your review again and approve this PR? Thanks

**Comment by DannyS03** [comment]
- verified on **qa**.

**Pull request merged**